### PR TITLE
Move from blue to OpenMDAO 2.0.0

### PR DESCRIPTION
**Pull request by kmarsteller**
Here's what we've got...suggestions welcome on README verbiage.

### CODE REVIEW

File: release_notes.txt [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        - LinearRunOnce
+        - DirectSolver
+        - PetscKSP
+        - ScipyIterativeSolver
+        - LinearUserDefined
+    - LineSearch
+        - AmijoGoldsetinLS
+        - BoundsEnforceLS
+- Components:
+    - MetModel
```
Comment (swryan) [inline release_notes.txt]: MetaModel

File: .travis.yml [hunk 2]
```diff
@@ -55,9 +55,7 @@ before_install:
 
 install:
 - conda install --yes python=$PY numpy==1.11.2 scipy nose sphinx mock swig
-- if [ "$OS" = "Linux" ]; then if [ "$PY" = "2.7" ]; then pip install https://openmdao.org/dists/pyoptsparse-1.0.0-cp27-none-linux_x86_64.whl;
-  elif [ "$PY" = "3.4" ]; then pip install https://openmdao.org/dists/pyoptsparse-1.0.0-py3-none-linux_x86_64.whl;
-  fi fi
```
Comment (swryan) [inline .travis.yml]: need pyoptsparse for linux too, right?

Comment (kmarsteller) [reply .travis.yml]: Fixed

File: release_notes.txt (hunk 1 above)
Comment (kmarsteller) [reply release_notes.txt]: fixed

**Pull request merged**